redlines/cli.py

```
@@ -18,64 +18,172 @@
+        "\n"
+        f"Cause: redlines v{version('redlines')} requires Python {requires_python}.\n"
+        "\n"
+        "To fix: Upgrade your Python version:\n"
+        "  # Check your current Python version\n"
+        "  python --version\n"
+        "\n"
+        "  # Install a newer Python version from python.org\n"
+        "  # Or use a version manager like pyenv:\n"
+        "  pyenv install 3.12\n"
+        "  pyenv global 3.12\n"
+    )
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)